### PR TITLE
Release 10-25-2023

### PR DESCRIPTION
**Pull request by nikhitasharma**
Releasing the below packages:
 - terra-collapsible-menu-view@6.83.0
  - terra-data-grid@1.0.0
  - terra-file-path@1.0.0
  - terra-folder-tree@0.1.1-alpha.0
  - @cerner/terra-framework-docs@1.43.0
  - terra-menu@6.79.0
  - terra-slide-panel-manager@5.83.0
  - terra-slider@1.0.1
  - terra-table@5.1.1
  - terra-tabs@7.11.1

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by JessieRandle — APPROVED** [review]

File: packages/terra-file-path/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 1.0.0 - (October 25, 2023)
+
+* Changed
+  * Minor dependency version bump
```
Comment (sdadn) [inline packages/terra-file-path/CHANGELOG.md:8]: If we're doing a full release, can we update the CHANGELOG so it's more reflective of the changes? I don't know what happened here but there are no details in this file at all.

I guess for simplicity, we can update this to:

```suggestion
  * Initial working component release.
```

File: packages/terra-file-path/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
+
 ## 0.3.0 - (October 17, 2023)
 
 * Inital stable release.
```
Comment (sdadn) [inline packages/terra-file-path/CHANGELOG.md:12]: Can we correct this to:
```suggestion
* Initial package release.
```

File: packages/terra-data-grid/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-data-grid",
   "main": "lib/index.js",
-  "version": "0.8.1",
+  "version": "0.9.0",
```
Comment (sdadn) [inline packages/terra-data-grid/package.json:4]: Should this be a 1.0 release?

cc: @JessieRandle , @cm9361

**Review by sdadn — CHANGES_REQUESTED** [review]

File: packages/terra-file-path/CHANGELOG.md (hunk 2 above)
Comment (nikhitasharma) [reply packages/terra-file-path/CHANGELOG.md]: Updated

File: packages/terra-data-grid/package.json (hunk 3 above)
Comment (nikhitasharma) [reply packages/terra-data-grid/package.json]: Updated terra-data-grid to release 1.0.0

**Review by sdadn — APPROVED** [review]

**Pull request merged**